core/src/main/java/org/apache/iceberg/view/BaseView.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.view;
```

Comment:
Packages in Iceberg are typically plural, like `actions`, `events`, `exceptions`, `expressions`, `metrics`, `transforms`, or `types`.

Reply:
are you suggesting to rename `org.apache.iceberg.view` to `org.apache.iceberg.views`?

Reply:
If we haven't already released with this name, then yes.

Reply:
I think it's too late to change package names because we released already with this naming